README.md

```diff
@@ -76,6 +76,31 @@ Results in:
 <script src="/js/jquery-[hash].js" async></script>
 ```
 
+You can also reference image paths via the `assetPath` helper. First, you must specify the
+path to your images via the `paths` option e.g:
+```javascript
+...
+
+var assets = require('connect-assets');
+
+app.use(assets({
+  paths: [
+    'assets/css',
+    'assets/js',
+    'assets/img'
+  ]
+}));
+```
+You can then use the `assetPath` helper in your Jade like so:
+```
+img(src="#{assetPath('image-name.png')}") 
```

Comment:
I’m no Jade expert, but you can axe the extra interpolation, right? 

```jade
img(src=assetPath('image-name.png'))
```

But then again, maybe it’s better as you’ve written, since it’s really clear what’s happening.